README.md

```diff
@@ -148,6 +149,20 @@ For example, if your project doesn't use patch version numbers, you can set `ver
 
 Release Drafter requires full write, because GitHub does not offer a limited scope for only writing releases. **Don't install Release Drafter to your entire GitHub account — only add the repositories you want to draft releases on.**
 
+## Replacers
+
+With the `replacers` option you can provide string replaces for your template after all other template information is finished being processed.
+
+```yml
+replacers:
+  - regex: /JENKINS-(\d+)/g
+    replace: https://issues.jenkins-ci.org/browse/JENKINS-$1
+  - search: myname
+    replace: My Name
+```
+
+Multiple replacers will be applied in order, so if your replacement string matches something in the second replacer, it'll get triggered again
```

Comment:
Let's remove this, in favour of the paragraph above at the start of the section?